pkg/test/framework/features/features.yaml

```diff
@@ -64,6 +64,7 @@ features:
     peer:
       secure-naming:
       trust-domain-validation:
+      trust-domain-validation-alias-client:
```

Comment:
trust-domain-alias-secure-naming  might be a better name.

Reply:
will do